helpers/getImage.js

```diff
@@ -1,23 +1,22 @@
 'use strict';
-
-var _ = require('lodash');
+const common = require('./lib/common.js');
 
 const factory = globals => {
     return function(image, presetName, defaultImageUrl) {
-        var sizeRegex = /^(\d+?)x(\d+?)$/g;
-        var settings = globals.getThemeSettings() || {};
-        var presets = settings._images;
-        var size;
-        var width;
-        var height;
-
-        if (!_.isPlainObject(image) || !_.isString(image.data) || image.data.indexOf('{:size}') === -1) {
+        const sizeRegex = /^(\d+?)x(\d+?)$/g;
+        const settings = globals.getThemeSettings() || {};
+        const presets = settings._images;
+        let size;
```

Comment:
Prefer one line for this: `let size, width, height;`

Reply:
ok, changed.